src/world/area_kpa/kpa_04/A4BF60.c

```diff
@@ -5,4 +5,30 @@ static char* N(exit_str_1) = "kpa_81";
 
 #include "world/common/UnkFunc11.inc.c"
 
-INCLUDE_ASM(s32, "world/area_kpa/kpa_04/A4BF60", func_802400BC_A4C01C);
+ApiStatus func_802400BC_A4C01C(Evt *script, s32 isInitialCall) {
+    PlayerStatus* playerstatus = &gPlayerStatus;
+    f32 sp24;
```

Comment:
Name this variable `dist`, see `npc_raycast_down_sides`'s signature for more info